docs/service-catalog/docs/050-gs-broker-registration.md

```diff
@@ -0,0 +1,67 @@
+---
+title: How to register a broker
+type: Getting Started
+---
+
+This Getting Started guide shows how to register a new broker in the Service Catalog. The broker can be either a Namespace-scoped ServiceBroker or a cluster-wide ClusterServiceBroker. Follow this guide to register a cluster-wide [UPS Broker](https://github.com/kubernetes-incubator/service-catalog/tree/master/charts/ups-broker) in the Service Catalog.
+
+## Prerequisites
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl)
+* [helm](https://github.com/helm/helm#install)
+
+## Steps
+
+1. Clone the [`service-catalog`](https://github.com/kubernetes-incubator/service-catalog) repository to your local machine:
```

Comment:
"Clone (...) to your local machine:" 
I don't think that "to your local machine:" brings any meaning or value. Where else would you clone the repository? `git clone` allows you to download to your machine only.